arxiv/identifier/__init__.py

```diff
@@ -25,23 +25,24 @@
 )
 
 OLD_STYLE_WITH_ARCHIVE = re.compile(
-    r'%s(?P<arxiv_id>(%s)\/\d{2}[01]\d{4}(v\d*)?)' % (_prefix, _archive),
```

Comment:
If the named groups (like `P<arxiv_id>`) are being removed from the regexes, then anything that uses them (see `parse_arxiv_id` below) should be updated to not use them. Personally, I find named groups more readable.

Reply:
Reverted this; added a function in ``arxiv.base.urls`` to remove the group names so that I can re-use them without breaking ``arxiv.identifier``.

Reply:
Oh, and now we have some tests in ``arxiv.identifier``!

Reply:
Thanks!